farcy/helpers.py

```diff
@@ -134,10 +193,17 @@ def __init__(self, repository=None):
         self.limit_users = None
         self.log_level = 'NOTSET'
         self.pr_issue_report_limit = 128
+        self.session = None
 
         self._dirty = set()
         self._config_path = os.path.join(CONFIG_DIR, 'farcy.conf')
 
+    def ensure_session(self):
```

Comment:
I think we could do away with this and make `session` a property of `Config` that returns the existing session if it's set. No need to update for this PR, however.